src/NuGet.Indexing/Handlers/LatestListedHandler.cs

```diff
@@ -105,5 +111,11 @@ private static bool GetListed(Document document)
             string listed = document.Get("Listed");
             return (listed == null) ? false : listed.Equals("true", StringComparison.InvariantCultureIgnoreCase);
         }
+
+        private static bool GetSemver2(Document document)
+        {
+            string semverLevel = document.Get("SemverLevel");
```

Comment:
Actually I think the best naming would be `GetIsSemVer2`.

`GetListed` should also be changed to `GetIsListed`.